challenge_01/base_solution.py

```diff
@@ -0,0 +1,103 @@
+from typing import List, Tuple
+
+import torch
+import torch.nn as nn
+import torch.utils.data as data
+
+
+class MultiLayerPerceptron(nn.Module):
+    """
+    Model class follow rules by nn.Module
+    I tried to fill class by glues I got.
+    Key trick is create hidden layers by blocks follow nn.Sequential
+    """
+
+    def __init__(
+        self,
+        layer_dim_list: List[int],
+        activation: nn.modules.activation,
```

Comment:
teď se spíš používá torch.relu(), atd...

nevím jestli je na to jiný typing  než Callable

Reply:
s týmto som si nevedel rady tak som tam hodil intuitivne čo prislo vhod podla triedy od kial pochádza aktivačná funkcia ale je pravda že tam môžeš dať potom aj niejaku vlastnu a už to bude držkovať, tak som to zmenil